src/Logic.hs

```diff
@@ -407,23 +419,44 @@ handleCommentAdded triggerConfig projectConfig prId author body state =
     -- a user has push access requires an API call.
     Just pr -> do
       let approvalType = parseMergeCommand triggerConfig body
-      isApproved <- if isJust approvalType
-        then isReviewer author
-        else pure False
+      isApproved <-
+        if isJust approvalType
+          then isReviewer author
+          else pure False
+      -- For now Friday at UTC+0 is good enough. 
+      -- See https://github.com/channable/hoff/pull/95 for caveats and improvement ideas.
```

Comment:
Can you open a new hoff issue about potentially making this more configurable (e.g. the timezone and the range)?
Then we'll also immediately have a good first issue for the next person joining :-)